Content.Server/Doors/Systems/FirelockSystem.cs

```diff
@@ -136,8 +142,17 @@ private void OnBeforeDoorOpened(EntityUid uid, FirelockComponent component, Befo
             // Give the Door remote the ability to force a firelock open even if it is holding back dangerous gas
             var overrideAccess = (args.User != null) && _accessReaderSystem.IsAllowed(args.User.Value, uid);
 
-            if (!this.IsPowered(uid, EntityManager) || (!overrideAccess && IsHoldingPressureOrFire(uid, component)))
+            var isPowered = this.IsPowered(uid, EntityManager);
```

Comment:
```suggestion
            var isPowered = IsPowered(uid, EntityManager);
```
`this.` can be removed. Seems to just be old code.

Reply:
So this (pun intended) one is interesting. It's not using `this` in the context of calling a class member function, but rather there's a `StaticPowerSystem.IsPowered `static function we're calling that takes a `this` ref as its first argument. Which I'll be honest I don't see that very often.